src/component/HorizontalFooter/index.js

```diff
@@ -1,110 +1,140 @@
 import React, { Component } from "react";
 import { Container, Row, Col } from "react-bootstrap";
 import "./style.css";
-import GenerateLogo from "../../assets/images/landingpage/affiliateorgintros/Generate Logo_Nav.png";
+import GenerateLogo from "../../assets/images/landingpage-v2/footerlogo.svg"; //"../../assets/images/landingpage-v2/footerlogo.svg";
 import InstaIcon from "../../assets/images/socialMediaIcons/Insta.png";
 import LinkedInIcon from "../../assets/images/socialMediaIcons/Linkedin.png";
 import FbIcon from "../../assets/images/socialMediaIcons/Facebook.png";
 import YoutubeIcon from "../../assets/images/socialMediaIcons/Youtube.png";
 import MosaicIcon from "../../assets/images/socialMediaIcons/Mosaic.png";
 import NortheasternIcon from "../../assets/images/socialMediaIcons/Northeaster.png";
 import Sherm from "../../assets/images/socialMediaIcons/Sherm.png";
+import { useNavigate } from "react-router-dom";
+
+const pages = [
```

Comment:
Aren't these somewhere else as well? Move to /shared?

Reply:
The icons? Assets is a partial shared